s3response/s3response.go

```diff
@@ -112,6 +112,10 @@ type Tagging struct {
 	TagSet  TagSet   `xml:"TagSet"`
 }
 
+type TaggingInput struct {
+	TagSet TagSet `xml:"TagSet"`
+}
+
 type DeleteObjects struct {
```

Comment:
Maybe we better remove Tagging struct above and instead start using this new TaggingInput?

Reply:
I think we want to keep with the
```
XMLName xml.Name `xml:"http://s3.amazonaws.com/doc/2006-03-01/ Tagging" json:"-"`
```
for the responses to keep in sync with the aws response
the bug in this one was when the client sent the xml tags without this namespace
the official request syntax also does show the namespace https://docs.aws.amazon.com/AmazonS3/latest/API/API_PutBucketTagging.html
```
PUT /?tagging HTTP/1.1
Host: Bucket.s3.amazonaws.com
Content-MD5: ContentMD5
x-amz-sdk-checksum-algorithm: ChecksumAlgorithm
x-amz-expected-bucket-owner: ExpectedBucketOwner
<?xml version="1.0" encoding="UTF-8"?>
<Tagging xmlns="http://s3.amazonaws.com/doc/2006-03-01/">
   <TagSet>
      <Tag>
         <Key>string</Key>
         <Value>string</Value>
      </Tag>
   </TagSet>
</Tagging>
```
but I guess not all clients will strictly follow this